project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=0.13.15
```

Comment:
agh, sorry. is it alright if I add `build.properties` to the `.gitignore`? IntelliJ seems dead-set on leaving these everywhere.